draft-ietf-quic-transport.md

```diff
@@ -2251,16 +2251,21 @@ transport parameter in the TLS handshake.
 Servers MAY communicate a preferred address of each address family (IPv4 and
 IPv6) to allow clients to pick the one most suited to their network attachment.
 
-Once the handshake is finished, the client SHOULD select one of the two
+Once the handshake is confirmed, the client SHOULD select one of the two
 server's preferred addresses and initiate path validation (see
-{{migrate-validate}}) of that address using the connection ID provided in the
-preferred_address transport parameter.
+{{migrate-validate}}) of that address using an active unused connection ID.
 
 If path validation succeeds, the client SHOULD immediately begin sending all
-future packets to the new server address using the new connection ID and
-discontinue use of the old server address.  If path validation fails, the client
-MUST continue sending all future packets to the server's original IP address.
+future packets to the new server and discontinue use of the old server address.
+If path validation fails, the client MUST continue sending all future packets to
+the server's original IP address.
 
+When the client finishes migrating to the preferred address, it retires the
```

Comment:
This is not currently required anywhere that I can find, do you want to make this a "SHOULD retire"?

Reply:
That's a keen observation. I had assume that we'd state that:
* a connection ID that was used on one path MUST NOT be used on a path created by migration
* a connection ID that becomes unusable SHOULD be retired

But surprisingly, we did not. As these requirements are common to ordinary connection migration and migration by preferred address, I've changed the sections that are shared by the two.

PTAL at commits 8201bd6 and dc1c5f7.